aries_cloudagent/config/argparse.py

```diff
@@ -545,6 +556,20 @@ def get_settings(self, args: Namespace) -> dict:
         settings = {}
         if args.external_plugins:
             settings["external_plugins"] = args.external_plugins
+
+        if args.plugin_config:
+            with open(args.plugin_config, "r") as stream:
+                plugins_conf = yaml.safe_load(stream)
+
+            if not settings.get("external_plugins"):
+                settings["external_plugins"] = []
+
+            settings["plugins_config"] = {}
+            for plugin, conf in plugins_conf.items():
+
+                settings["external_plugins"].append(plugin)
+                settings["plugins_config"][plugin] = conf
+
```

Comment:
I strongly recommend that we don't allow loading of configs from the plugin configuration. If we consider, for example, the typical `setup.cfg` file or `pyproject.toml`, you can have a section for a "tool" with configuration set but unless you load the tool into your virtual environment and execute it, that configuration has no effect, positive or negative. I think we should follow this pattern. If configuration is present for a plugin but the plugin isn't loaded, then the configuration is just ignored.

Reply:
```suggestion
            settings["plugin_config"] = plugin_config
```

Reply:
We should favor `plugin_config` instead of `plugins_config`